Mobile menu

ragangithub opened the pull request:
# In this branch
- added JS file
- updated CSS
- updated HTML
- added JavaScript linter

CollinsTatang reviewed (commented):
Hi @ragangithub, 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a proffessional README file.

- [ ] Please remove the Microverse logo from your README file and add yours if you have one.
- [ ] Please your commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Kindly correct that in your next commit massage.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

ragangithub commented:
- removed Microverse logo from README file
- removed unnecessary comments from HTML and CSS file

zieeco commented on script.js:
```diff
@@ -0,0 +1,11 @@
+const menu = document.querySelector('.menu');
+const closee = document.querySelector('.close');
+const navLinks = document.querySelector('.nav-links');
```
- [ ]  Hey, here you're targetting the container of each nav link, instead of all the links, so that when a user clicks on any link, it should take them to that section of your portfolio and the menu should disappear too. Currently, this is not working as expected. Please, kindly improve on this to meet the project requirement.

zieeco reviewed (changes requested):
Hi @Team,

<img height="230" src="https://media0.giphy.com/media/7hyhCyGbFX50Vc9jah/giphy.gif"/>



Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!

## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ragangithub commented:
- Used the correct background color
- Used the correct font weight on the links and the cancel button
- Added the missing blur effect on the background
- Added the functionality when a user clicks on any link, it takes them to part of a section of the portfolio

Gambit142 reviewed (approved):
Hi @ragangithub @MarcoDiaz2000 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Links in the mobile menu direct users to the correct part of the page ✔️ 
- Use of JavaScript to make navigation successful ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
